webapp/components/admin_console/logs.jsx

```diff
@@ -26,6 +26,12 @@ export default class Logs extends React.Component {
         AsyncClient.getLogs();
     }
 
+    componentDidUpdate() {
+        // Scroll Down to get the latest logs
+        var node = ReactDOM.findDOMNode(this.refs.logPanel);
```

Comment:
Minor, but you should be able to just use `var node = this.refs.logPanel`. ReactDOM is scheduled to be deprecated so if you could switch it over that would be great